en/developer/reference/articles/02-workflow/05-notification-templates.markdown

```diff
@@ -0,0 +1,287 @@
+# Workflow Notification Templates
+
+There are a number of variables auto-injected into the workflow notification
+context. These are handy to use in your workflow notification templates.
```

Comment:
I'd like to add this:

> The exact variable set injected into your workflow context depends on several things: 
> Things 1 
> Thing 2
> Thing 3

Basically just give the reader the idea about why it won't be the same for every notification.
cc @rafaprax

Reply:
@rafaprax You can ignore this one, I think it's enough to say

> The precise variables available change depending on the workflow definition details and the asset being operated on in the workflow.